example/spec/loadTest/ingestLoad_queueGranPassThruSF.js

```diff
@@ -0,0 +1,253 @@
+      `\n___ ${incompletedGranules.length}/${inputPayload.granules.length} Granules incompleted by workflow ${nthWorkflow} ___`);
+
+    incompletedGranules.map((granuleError) =>
+      console.log(redConsoleLog(),
+        `\n${JSON.stringify(granuleError.granuleId)} - ${granuleError.err}`));
+  }
+};
+
+describe('The Granule Ingest Load Test ', () => {
+  beforeAll(async () => {
+    if (!Number.isFinite(Number(granuleCountPerWorkflow)) || granuleCountPerWorkflow > 450) {
+      beforeAllFailed = `===== beforeAll() - Invalid input for number of granules per Workflow detected - ${granuleCountPerWorkflow} =====`;
+      throw new Error(beforeAllFailed);
+    } else {
+      try {
+        config = await loadConfig();
+
+        process.env.GranulesTable = `${config.stackName}-GranulesTable`;
+        process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
+        process.env.CollectionsTable = `${config.stackName}-CollectionsTable`;
+
+        await pMap(
+          new Array(totalWorkflowCount).fill().map((_, index) => index + 1),
+          async (index) => {
+            await batchGranulesProcessing(index);
+          },
+          { concurrency: 8 }
+        );
+      } catch (error) {
+        beforeAllFailed = `===== beforeAll() failed =====\n ${error}`;
+        throw new Error(beforeAllFailed);
+      }
+    }
+  });
+
+  afterAll(async () => {
+    // clean up stack state added by test
+    await Promise.all(
+      totalInputPayloads.map(async (inPayload) =>
+        await cleanupLoadTestGranules(config.stackName, inPayload.granules))
+    );
+
+    console.log(blueConsoleLog(), '\n===== Delete lambdaOutputPayload ExecutionArns =====');
+    queueGranulesChildExecutionArns.forEach((lambdaOutputPayload) => {
+      lambdaOutputPayload.running.forEach((childExecutionArn) => {
+        deleteExecution({
+          prefix: config.stackName,
+          executionArn: childExecutionArn,
+        });
+        // console.log(blueConsoleLog(), `${JSON.stringify(childExecutionArn)}`);
+      });
+    });
+
+    if (queueGranulesExecutionArns !== undefined) {
+      console.log(blueConsoleLog(), '\n===== Delete queueGranules ExecutionArns =====');
+      queueGranulesExecutionArns.forEach((qGranulesExecutionArn) => {
+        deleteExecution({
+          prefix: config.stackName,
+          executionArn: qGranulesExecutionArn,
+        });
+        console.log(blueConsoleLog(), `${JSON.stringify(qGranulesExecutionArn)}`);
+      });
+    }
+
+    await Promise.all(
+      testSuffixes.map(async (testSuffix) => {
+        await cleanupCollections(
+          config.stackName,
+          config.bucket,
+          collectionsDir,
+          testSuffix
+        );
+        cleanupProviders(
+          config.stackName,
+          config.bucket,
+          providersDir,
+          testSuffix
+        );
+      })
+    );
+  });
+
+  it('writes to database the expected number of granules with status completed', () => {
```

Comment:
We'd talked about adding additional assertions here. I think using QueueGranules might be alright if we're willing to make do with the limitations in using a workflow (e.g. we have to test the workflow to test that the rest of the framework works) but I'm still hesitant about what we're actually testing.

This test seems to say that "QueueGranules successfully wrote the expected number of granules". That might be the best we can do but it feels like we're just asserting that the workflow works. Any failures in DB writes, queuing, etc. would be a side effect and not explicitly tested.

I'd like other opinions here. @Jkovarik? You had thoughts about this during PI planning.

Reply:
Put another way, what is the question these tests are attempting to answer? How many granules can we ingest using Cumulus before it blows up? Can we answer that?

Reply:
The question these tests are attempting to answer is, Can the system still handle ingestion of a large specified amount of granules after any code change? And they did/should.

Reply:
I think there's a legit discussion here.  

I commented in the review https://github.com/nasa/cumulus/pull/3073#discussion_r975772654 re: it, but I'm not as concerned about some of the data integrity concerns we've uncovered elsewhere as those were logic failures.  We do need to be careful we're sure all records make it in, and not rely on counts, but the purpose of this test should be narrowly constrained to load related concerns.    Happy to discuss further what I mean by that.